src/test/kotlin/no/nav/familie/dokument/storage/TokenValidationContextHolderExtentionTest.kt

```diff
@@ -0,0 +1,69 @@
+package no.nav.familie.dokument.storage
+
+import io.mockk.every
+import io.mockk.mockk
+import no.nav.security.token.support.core.context.TokenValidationContextHolder
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+internal class TokenValidationContextHolderExtentionTest {
+
+    private val tokenValidationContextHolderMock = mockk<TokenValidationContextHolder>()
+
+    @BeforeEach
+    internal fun setUp() {
+        every { tokenValidationContextHolderMock.hentFnr() } returns "test_fnr"
+    }
+
+    @Test
+    internal fun `Mappenavn for bruker skal være 44 lang`() {
+        assertThatDigestHarLengde44("12345678911",
```

Comment:
Fint å sjekke lengden!
Kanskje kan ha med expected også, hva output for de ulike er? 
Eks så kommer alle avsluttes med `=`, men virker vel som att gcp støtter det

Reply:
Ble litt usikker på det med lengdebegrensning når jeg leste gcp spec. Bucket navn kan ikke være mer en 63 tegn, mens object navn kan være lange. Jeg tror dirnavn sorterer under object, mens bøttene vår heter familie-dokument-dev og familie-dokument, men her kan jeg ta feil.

Reply:
Det er riktig